vnext/Microsoft.ReactNative/Modules/AppStateModule.h

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#pragma once
+
+#include <NativeModules.h>
+#include <winrt/Windows.ApplicationModel.h>
+#include <winrt/Windows.Foundation.h>
+
+namespace Microsoft::ReactNative {
+
+REACT_MODULE(AppState)
+struct AppState : public std::enable_shared_from_this<AppState> {
+  REACT_INIT(Initialize)
+  void Initialize(winrt::Microsoft::ReactNative::ReactContext const &reactContext) noexcept;
+
+  REACT_METHOD(getCurrentAppState)
+  void getCurrentAppState(
+      std::function<void(React::JSValue const &)> const &success,
+      std::function<void(React::JSValue const &)> const &error) noexcept;
+
+  REACT_METHOD(addListener)
+  void addListener(std::string eventName) noexcept;
```

Comment:
>std::string [](start = 19, length = 11)

Consider using && or const&.
In Office we usually do not pass types with non-trivial destructors by value because it generates more code.

Reply:
The spec checker doesn't seem to like const&

Reply:
I must fix the checker then. It sounds as a bug.

---
In reply to: [429378582](https://github.com/microsoft/react-native-windows/pull/4991#discussion_r429378582) [](ancestors = 429378582)

Reply:
#4992 filed

Reply:
Used && for now, which does work.